deploy/operator.yaml

```diff
@@ -191,7 +191,7 @@ spec:
         - name: SRIOV_CNI_IMAGE
           value: quay.io/kubevirt/cluster-network-addon-sriov-cni:v1.1.0-1.git9e4c973
         - name: NMSTATE_HANDLER_IMAGE
-          value: quay.io/nmstate/kubernetes-nmstate-handler:v0.3.0
+          value: quay.io/nmstate/kubernetes-nmstate-handler:v0.6.0
```

Comment:
Unfortunately, it's the `hack/build-manifests.sh` script in conjunction with `manifest-templator.go` that build these files. If you want this change to survive the next time someone builds manifests, you will need to:

1. update it in cluster-network-addons-operator [here](https://github.com/kubevirt/cluster-network-addons-operator/blob/21f2cbaef0e6ef8c292c7c70070fc86d9b6e92db/pkg/components/components.go#L28)
1. bump vendored version of cluster-network-addons-operator in this project
1. rebuild manifests with `hack/build-manifests.sh`

Reply:
I have the cluster-network-addons-operator here https://github.com/kubevirt/cluster-network-addons-operator/pull/202

Reply:
@qinqon looks like the PR merged.  When the CNA creates a new release we can pick it up.

Reply:
@rthallisey We need also this https://github.com/kubevirt/cluster-network-addons-operator/pull/206 to have new releas of addons operator